graf2d/gpadv7/v7/inc/ROOT/RCanvas.hxx

```diff
@@ -105,6 +108,9 @@ public:
    /// update drawing
    void Update(bool async = false, CanvasCallback_t callback = nullptr);
 
+   /// Run canvas functionality for given time
+   void Run(double tm = 0.);
```

Comment:
Could the unit (seconds?) be documents? What does "run canvas" mean - how do you "run a canvas"? Could you elaborate just a tiny bit in the doc?

Reply:
Is this running the event loop in the caller thread? If so, how can you "guarantee" the time is obeyed? Or is this a minimum / estimate?

Reply:
Yes, units are seconds. There is no guarantee - it is minimal time how long `Run()` method will be invoked. 

I add this info with example code in implementation file for `RWebWindow::Run()` method.